packages/wasmkit/src/lib/deploy/contract.ts

```diff
@@ -264,30 +264,36 @@ export class Contract {
     return instantiateInfo;
   }
 
-  async queryMsg (msgData: Record<string, unknown>): Promise<any> { // eslint-disable-line  @typescript-eslint/no-explicit-any
+  async queryMsg (msgData: Record<string, unknown>, skipLogs?: boolean): Promise<any> { // eslint-disable-line  @typescript-eslint/no-explicit-any
     if (this.contractAddress === "mock_address") {
       throw new WasmkitError(ERRORS.GENERAL.CONTRACT_NOT_INSTANTIATED, {
         param: this.contractName
       });
     }
     // Query the contract
-    this.printLoadingAnimation(`[${chalk.gray("wasmkit")}] ${chalk.green("INF")} Querying ${this.contractAddress} => ${Object.keys(msgData)[0]}`);
-
+    if (!skipLogs) {
```

Comment:
This should be if `skipLogs === true` then print the logs

Reply:
But sir if we want logging to be enabled by default, then I believe it should be ```if(skipLogs===false)``` as by default skipLogs is now initialized as ```skipLogs: boolean = false```. (As when skipLogs is true, we will be skipping logs, and when false we will be printing logs)

Reply:
Yeah, so just set the skipLogs param as true by default

Reply:
ok sir

Reply:
I have renamed the skipLogs param to logFlag as it seemed more appropriate(to minimize confusion)